staker-rewards/lib/context.js

```diff
@@ -4,10 +4,12 @@ import WebsocketSubprovider from "web3-provider-engine/subproviders/websocket.js
 import Subproviders from "@0x/subproviders"
 
 import Cache from "./cache.js"
-import { Contract, getDeploymentBlockNumber } from "./contract-helper.js"
+import {Contract, getDeploymentBlockNumber} from "./contract-helper.js"
 
+import TokenStakingJson from "@keep-network/keep-core/artifacts/TokenStaking.json"
 import BondedECDSAKeepFactoryJson from "@keep-network/keep-ecdsa/artifacts/BondedECDSAKeepFactory.json"
 import BondedECDSAKeepJson from "@keep-network/keep-ecdsa/artifacts/BondedECDSAKeep.json"
+import TBTCSystemJson from "@keep-network/tbtc/artifacts/TBTCSystem.json"
```

Comment:
Shouldn't we depend on `@keep-network/keep-core` and `@keep-network/tbtc` in the `package.json` file?